src/sentinel.c

```diff
@@ -4153,6 +4157,85 @@ void sentinelRoleCommand(client *c) {
     dictReleaseIterator(di);
 }
 
+/* SENTINEL GET <mastername> <option> */
```

Comment:
@hwware Is there real value in this filtering?
I wonder if it won't be simpler to just have `SENTINEL GET` to get all masters, or `SENTINEL GET <master>` to get a specific one. This will also solve the (rare?) ambiguity around a master named `all`.